NanoServerPackage/NanoServerPackage.psm1

```diff
@@ -2240,155 +2206,6 @@ function Find-Package
         }
     }
 
-    <# Commented out because we are not handling source yet
-    # no source given then search online
```

Comment:
We should add it back whenever required.

---
In reply to: [82315630](https://github.com/OneGet/NanoServerPackage/pull/6#discussion_r82315630) [](ancestors = 82315630)